webpack.config.babel.js

```diff
@@ -42,7 +42,7 @@ const config = {
   },
   plugins: [
     new Dotenv({
-      safe: true,
+      safe: false,
```

Comment:
@aaronsekisambu What was the reason behind changing `safe: true` to **_false_**?

Reply:
@avpaul when safe is set to true you get ```javascript
.env.sample
```

complain the first time you check into any bracnch, check [here](https://github.com/mrsteele/dotenv-webpack#properties)